plugin/pkg/auth/authenticator/token/oidc/oidc.go

```diff
@@ -155,8 +156,19 @@ func (a *OIDCAuthenticator) AuthenticateToken(value string) (user.Info, bool, er
 		username = fmt.Sprintf("%s#%s", a.clientConfig.ProviderConfig.Issuer, claim)
 	}
 
-	// TODO(yifan): Add UID and Group, also populate the issuer to upper layer.
-	return &user.DefaultInfo{Name: username}, true, nil
+	// TODO(yifan): Add UID, also populate the issuer to upper layer.
+	info := &user.DefaultInfo{Name: username}
+
+	if a.groupsClaim != "" {
+		groups, ok, err := claims.StringsClaim(a.groupsClaim)
```

Comment:
how does this handle a missing claim? I'd expect that to result in an empty groups list, not an auth error